rasa/utils/tensorflow/models.py

```diff
@@ -109,12 +161,21 @@ def fit(
                         epoch_batch_size,
                         False,
                     )
-                    val_results = self._get_metric_results(prefix="val_")
+                    step = epoch if self.tensorboard_log_on_epochs else training_steps
+                    val_results = self._get_metric_results(
```

Comment:
Just tried it again, setting `tensorboard_log_level` to `minibatch` makes training logs at minibatch level but test logs at epoch level because of this function is called just once? Just confirming.

Reply:
We should make it consistent actually, if it is set to `minibatch` then both train and test logging should be at minibatch level. I tested with this config - 
```
- name: DIETClassifier
    epochs: 75
    batch_size: 2
    tensorboard_log_directory: "./tb_logs"
    tensorboard_log_level: "minibatch"
    evaluate_on_number_of_examples: 20
```
Test logs just have one record after every `evaluate_after_every_epochs` which is not possible because there should `20/2` validation minibatches.

Reply:
updated the code